src/pages/de/migrate.md

```diff
@@ -0,0 +1,687 @@
+---
+setup: | import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
```

Comment:
According to the log, I think a line break should be added after the vertical bar like this:

```suggestion
setup: | 
    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
```